### DISCUSSION TITLE
Problem running the configuration

### DISCUSSION THREAD

**Discussion by PA3CCE** [arkypita/LaserGRBL#2099] [Q&A]
Hi all,
I tried to adapt the configuration of my Kingroon KP3s Pro S1 by using LaserGRBL. 
During the process I seem to have done something wrong,  now I get a blank window  and a message "Error in loading config", when I click "GRBL/GRBL Configure" .
I understand that the config is stored into RAM of the  device and it looks that it is corrupted or even deleted , **can it be repared** ?
If so where to find the original config file and what is the procedure to reflash it ??
Your reaction will be appriciated.
Jos

**Comment by Tribo-2dot0** [comment]
Tente aqui:
https://kingroon.com/blogs/downloads/kingroon-kp3s-pro-stock-firmware-to-download

**Comment by PA3CCE** [comment]
Hello Trobo,
Thank you so mucch for the reply and the link.
Yes I do have that firmware indeed.
What I tried was the following:
I deleted the LaserGRBL software and cleaned my system (Windows1) the registry included.
Then I re-installed LaserGRBL and reflashed the firmware you referred to (ROBIN.NANO.bin and robin.nano.cfg.txt) 
into the KPS3S pro S1.
The result was still the same : The config window still is empty when I click "GRBL/GRBL config"
EEPROM on mainboard failure ???   It worked well in the beginning!
Regards 
Jos

**Comment by arkypita** [comment]
What if you type "$$" + hit enter in the "type gcode here" box?

**Comment by PA3CCE** [comment]
@ arkypita
Thank you for the info, I will do and report back !
Cheers
Jos

**Comment by PA3CCE** [comment]
@ arkypita
Very sorry, where to find that "type gcode here" box, can`t find it in LaserGRBL.

**Reply by Tribo-2dot0** [reply]
![Captura de tela 2023-07-16 114705](https://github.com/arkypita/LaserGRBL/assets/83982525/544ec0f7-301c-4bf2-bc63-9c7c0411ed74)

**Comment by PA3CCE** [comment]
@ Tribo-2dot0 and Arkypita
Thank you very much Tribo, now that it's there I can see it but it's **so** much grayed out that it is almost nimpossible to see it with my old eyes!.
When I enter the $$ command I see a lot of configuration settings indeed!
However the command I want to change "Laser on" is not present there and is set to 0 on my machine, 
I have added a lasermodule to the machine and  **there** I need to set "Laser on" to 1, when I  want to use it i.s.o the extruder.
That is my problem!
Regards,
Jos
